### PR TITLE
Shrilekha

### PR DESCRIPTION
**Pull request by Shrilekha68**
changes

### CODE REVIEW

**Review by HarishDass — APPROVED** [review]

**Review by aarthi1246 — APPROVED** [review]

File: src/app/attendance/attendance.component.html [hunk 1]
```diff
@@ -1 +1,5 @@
 <p>attendance works!</p>
+
+
+
+
```
Comment (bharathidhasan2002) [inline src/app/attendance/attendance.component.html]: Remove the spaces

File: src/app/employee-details/bar-chart/bar-chart.component.css [hunk 2]
```diff
@@ -0,0 +1,4 @@
+h3{
```
Comment (bharathidhasan2002) [inline src/app/employee-details/bar-chart/bar-chart.component.css]: use class or Id

File: src/interface/dataa.ts [hunk 3]
```diff
@@ -0,0 +1,36 @@
+export interface Dataa {
```
Comment (bharathidhasan2002) [inline src/interface/dataa.ts]: change naming convention

File: src/index.html [hunk 4]
```diff
@@ -6,6 +6,7 @@
   <base href="/">
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <link rel="icon" type="image/x-icon" href="favicon.ico">
+  <link rel="stylesheet" href="http://unpkg.com/primeflex@^3/primeflex.css">
```
Comment (bharathidhasan2002) [inline src/index.html]: don't written the index.html

File: src/app/employee-details/bar-chart/bar-chart.component.ts [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+  styleUrls: ['./bar-chart.component.css'],
+})
+export class BarChartComponent {
+  chartdata: any = [];
+  result: any;
+  Count: any;
+  constructor(private service: DataService) {}
+  ngOnInit() {
+    this.service.getdata().subscribe((res: any) => {
+      this.result = res;
```
Comment (bharathidhasan2002) [inline src/app/employee-details/bar-chart/bar-chart.component.ts]: don't use unwanted variable

File: src/app/employee-details/bar-chart/bar-chart.component.ts [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
+  templateUrl: './bar-chart.component.html',
+  styleUrls: ['./bar-chart.component.css'],
+})
+export class BarChartComponent {
+  chartdata: any = [];
+  result: any;
+  Count: any;
+  constructor(private service: DataService) {}
+  ngOnInit() {
+    this.service.getdata().subscribe((res: any) => {
```
Comment (bharathidhasan2002) [inline src/app/employee-details/bar-chart/bar-chart.component.ts:18]: can you use the function method

**Pull request closed without merging**